src/test/java/com/nerodesk/om/aws/AwsAttributesTest.java

```diff
@@ -0,0 +1,138 @@
+
+import com.jcabi.s3.mock.MkOcket;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.attribute.BasicFileAttributes;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Tests for {@link AwsAttributes}.
+ *
+ * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.comm)
+ * @version $Id$
+ * @since 0.3.30
+ * @todo #210:30min when the issue jcabi-s3:25 see
+ *  https://github.com/jcabi/jcabi-s3/issues/25 will be resolved
+ *  please uncomment the tests below. And make sure that all test
+ *  are working properly.
+ */
+public final class AwsAttributesTest {
+    /**
+     * AwsAttributes can return correct the doc size.
+     * @throws IOException If unsuccessful.
+     */
+    @Test
+    @Ignore
+    public void returnsCorrectSize() throws IOException {
+        final File file = AwsAttributesTest.createFile(".tmp");
+        MatcherAssert.assertThat(
+            new AwsAttributes(
+                new MkOcket(
+                    file,
+                    AwsAttributesTest.class.getSimpleName(),
+                    "size"
+                )
+            ).size(),
+            Matchers.equalTo(file.length())
+        );
+    }
+
+    /**
+     * AwsAttributes can return correct the doc creation date.
+     * @throws IOException If unsuccessful.
+     */
+    @Test
+    @Ignore
+    public void returnsCorrectDate() throws IOException {
+        final File file = AwsAttributesTest.createFile(".json");
+        MatcherAssert.assertThat(
+            new AwsAttributes(
+                new MkOcket(
+                    file,
+                    AwsAttributesTest.class.getSimpleName(),
+                    "date"
+                )
+            ).created().getTime(),
+            Matchers.equalTo(
+                Files.readAttributes(
+                    Paths.get(file.toURI()),
+                    BasicFileAttributes.class
+                ).creationTime().toMillis()
+            )
+        );
+    }
+
+    /**
+     * AwsAttributes can return correct the doc type.
+     * @throws IOException If unsuccessful.
+     */
+    @Test
+    @Ignore
+    public void returnsCorrectType() throws IOException {
+        final File file = AwsAttributesTest.createFile(".xml");
+        MatcherAssert.assertThat(
+            new AwsAttributes(
+                new MkOcket(
+                    file,
+                    AwsAttributesTest.class.getSimpleName(),
+                    "type"
+                )
+            ).type(),
+            Matchers.equalTo("application/xml")
+        );
+    }
+
+    /**
+     * Creates temporary file.
+     * @param extension The file extension
+     * @throws IOException In case of failure.
+     * @return The temporary file.
+     */
+    private static File createFile(final String extension) throws IOException {
```

Comment:
@dmzaytsev please use `TemporaryFolder` form junit for this (see e.g. `AwsDocsTest` how it is done)